recipes/sqlpp11/all/test_package/CMakeLists.txt

```
@@ -1,10 +1,11 @@
-cmake_minimum_required(VERSION 2.8.12)
-
-project(PackageTest)
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGETS)
```
Not really for conan targets, but to avoid to globally include headers folders: see https://github.com/conan-io/conan-center-index/pull/2669#discussion_r478446708

---

Indeed!